### ISSUE TITLE
EHAM doesn't work

### ISSUE THREAD

**Issue by DennisPlant** [zlsa/atc#754]
it just keeps loading? anyone have the same prolem?

**Comment by georgeenciu** [comment]
Same problem here

**Comment by willfrd** [comment]
Same

**Comment by n8rzz** [comment]
@erikquinn this fails in v3.1.0 due to a `_comment` key within the stars definition.

In v3.2.0 that has been addressed, however, this still fails from within the `PositionModel`.
```javascript
PositionModel.calculatePosition = (coordinates, referencePostion, magneticNorth) => {
    if (!coordinates || !referencePostion || !magneticNorth) {
        throw new TypeError('Invalid parameter. PositionModel.getPosition() requires coordinates, referencePostion ' +
            'and magneticNorth as parameters');
    }

    // ...
```

magneticNorth is being evaluated as falsey (magneticNorth = 0) thus, it throws.

**Comment by n8rzz** [comment]
@DennisPlant @georgeenciu @willfrd The fix is in PR.  When you have a minute, could you please take a look at the review app and make sure everything is in working order?  https://atc-dev-pr-187.herokuapp.com/

**Comment by willfrd** [comment]
Working, @n8rzz thanks for the fix! :)

**Comment by n8rzz** [comment]
:shipit: 

Glad to hear it!  This fix will be included in the upcoming v3.2.0 release

**Comment by erikquinn** [comment]
Resolved by #755. Closing.
